prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -12,6 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+resource "google_monitoring_alert_policy" "ci-absent" {
+  for_each     = var.prow_components
+  project      = var.project
+  display_name = "ci-absent-${each.key}"
+  combiner     = "OR" # required
+
+  conditions {
+    display_name = "ci-absent-${each.key}"
+
+    condition_monitoring_query_language {
+      duration = "1800s"
+      query    = <<-EOT
+      fetch k8s_container
+      | metric 'kubernetes.io/container/restart_count'
+      | filter
+          (resource.container_name == '${each.key}' && resource.namespace_name == '${each.value.namespace}')
+      | align rate(5m)
```

Comment:
Rate is going to return a value per second rather than the increase in value over the time period. e.g if there are 5 restarts in the last 5mins this will return 1/60, not 5.

Reply:
ack, moved away from using rate